### PR TITLE
4.6 release blog post

### PR DESCRIPTION
**Pull request by jenweber**
All team members and community members are encouraged to help write content for the release blog post.

## Sections to write content for:

- [x] ember.js
- [x] data
- [x] cli

## Get reviews

- [x] ember.js
- [x] data
- [x] cli

### CODE REVIEW

File: content/ember-4-6-released.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+date: 2022-07-13T21:00:00.000Z
+tags:
+  - releases
+  - '2022'
+  - version-4-x
+---
+
+Today the Ember project is releasing version 4.6 of Ember.js, Ember Data, and Ember CLI. <!-- Block start: Uncomment if an LTS candidate --><!--This release of Ember.js is an LTS (Long Term Support) candidate. LTS candidates prioritize stability over the addition of new features, and have an extended support schedule.--><!-- Block end -->
+
+This release kicks off the 4.6+0.1 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
```
Comment (kategengler) [inline content/ember-4-6-released.md]:
```suggestion
This release kicks off the 4.7 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
```

File: content/ember-4-6-released.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 4.6
+
+Ember.js 4.6 is an incremental, backwards compatible release of Ember with bug fixes, performance improvements, and minor deprecations.
+
+#### Bug Fixes
+
+Ember.js 4.6 introduced 0 bug fixes.
```
Comment (kategengler) [inline content/ember-4-6-released.md]: Two bugfixes - [#20125](https://github.com/emberjs/ember.js/pull/20125) [BUGFIX] Replace deprecated substr() method with substring() method.
and - [#20120](https://github.com/emberjs/ember.js/pull/20120) [BUGFIX] Adjust uniqueId() implementation to only generate valid selectors.

File: content/ember-4-6-released.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+#### Features
```
Comment (kategengler) [inline content/ember-4-6-released.md]: No features, no deprecations

File: content/ember-4-6-released.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+Consider using the [ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow) addon if you would like to upgrade your application without immediately addressing deprecations.
+
+<!-- Block end -->
+
+For more details on changes in Ember.js 4.6, please review the [Ember.js 4.6.0 release page](https://github.com/emberjs/ember.js/releases/tag/v4.6.0).
+
+---
+
+## Ember Data
+
```
Comment (kategengler) [inline content/ember-4-6-released.md:56]: We'll be releasing with ember-data 4.4 again

**Review by kategengler — CHANGES_REQUESTED** [review]

File: content/ember-4-6-released.md (hunk 2 above)
Comment (jenweber) [reply content/ember-4-6-released.md]: awesome, thank you! these are now incorporated.

File: content/ember-4-6-released.md (hunk 3 above)
Comment (jenweber) [reply content/ember-4-6-released.md]: Got it, thanks!

File: content/ember-4-6-released.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+* [#8025](https://github.com/emberjs/data/pull/8025) Fix `PromiseProxy` Deprecation for Save ([@runspired](https://github.com/runspired))
+* [#7902](https://github.com/emberjs/data/pull/7902) test for async hasMany does not trigger relationship fetch
+* Pass correct args to `keyFor<Attribute|Relationship>` when normalizing
+* [#7945](https://github.com/emberjs/data/pull/7945) fix for an issue with computed chains not updated for async hasMany loading
+
+#### Features
+
+Ember Data 4.5 introduced improvements to build size.
+
+Users of solely the core of Ember Data may expect their builds to be 35-50% smaller. This is due to most of the legacy support for `@ember-data/model` having now found its final location at home in that package instead of within the `@ember-data/store package`. The size of `@ember-data/store` is reduced from ~16kb compressed to ~9.5kb compressed, potentially a little less depending on what deprecations are resolved and what other packages are present. We expect the core to continue to shrink as we complete the final stages of removing `InternalModel`, complete implementations for recently accepted deprecation RFCs, and introduce `RecordDataV2`.
```
Comment (jenweber) [inline content/ember-4-6-released.md]: @runspired can you help scope/qualify this statement?

> Users of solely the core of Ember Data may expect their builds to be 35-50% smaller.

What does this mean? Is it that a specific ember-data package is 35-50% smaller? Thanks!

Comment (runspired) [reply content/ember-4-6-released.md]: Broadly speaking this means the store package. It currently also means ember-inflector as that's still a required dependency though it being required has been RFCs to deprecate. Notably users of ember-m3 tend to use only the store package.

File: content/ember-4-6-released.md [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+
+Ember.js 4.6 introduced 2 bug fixes.
+
+- [Replace deprecated `substr()` method](https://github.com/emberjs/ember.js/pull/20125) with `substring()` method
+- [Adjust `uniqueId()` implementation](https://github.com/emberjs/ember.js/pull/20120) to only generate valid selectors. [`querySelector`](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector) does not work if the first character is a number, so
+now we ensure that the first character is a letter.
+
+#### Features
+
+Ember.js 4.6 introduced 0 features.
```
Comment (kategengler) [inline content/ember-4-6-released.md:45]:
```suggestion
Ember.js 4.6 introduced 0 features.

```

**Review by kategengler — APPROVED** [review]

**Review by wycats — APPROVED** [review]
Looks great to me.

File: content/ember-4-6-released.md (hunk 5 above)
Comment (jenweber) [reply content/ember-4-6-released.md]: incorporated, thank you

**Pull request merged**